content/docs/reference/spec/migration/platform-api-0.11-0.12.md

```diff
@@ -59,9 +59,30 @@ In Platform 0.12 extensions can be used to extend not only build-time base image
 
 TODO
 
-### OCI layout is a supported export format
+### OCI layout is a supported export format (experimental)
 
-TODO
+In Platform 0.12 a new capability to [export an application images on disk in OCI format](https://github.com/buildpacks/rfcs/blob/main/text/0119-export-to-oci.md) was added.
+
+Platform must take care of preparing a [local storage](https://github.com/buildpacks/rfcs/blob/main/text/0119-export-to-oci.md#how-it-works) on disk where images are being saved in OCI layout format and passing it through to the lifecycle.
+
+#### Lifecycle phases affected
+
+The lifecycle phases affected by this new behavior are:
+- [Analyze](https://buildpacks.io/docs/concepts/components/lifecycle/analyze/)
+- [Restore](https://buildpacks.io/docs/concepts/components/lifecycle/restore/)
+- [Export](https://buildpacks.io/docs/concepts/components/lifecycle/export/)
+- [Create](https://buildpacks.io/docs/concepts/components/lifecycle/create/)
+
+#### Before executing the lifecycle
+
+Input images required by any phase, like the `run-image`, must be saved on disk in OCI layout format in the local storage previously defined following the 
```

Comment:
Is it just run image and previous image? We don't allow cache image in OCI layout do we?